tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -280,6 +284,89 @@ internal void UpdateAddedProbeInstrumentations(IReadOnlyList<ProbeDefinition> ad
             }
         }
 
+        private static string[] SafeParseSignature(string signature)
+        {
+            if (string.IsNullOrEmpty(signature))
+            {
+                return null;
+            }
+
+            try
+            {
+                return ParseSignature(signature);
+            }
+            catch (Exception e)
+            {
+                Log.Error(e, "Error parsing the signature {Signature}. Fall back to empty signature", signature);
+                return null;
+            }
+        }
+
+        private static string[] ParseSignature(string signature)
```

Comment:
IMO this method should 100% have a giant pile of unit tests 😄 It's trivially easy to test it by giving known inputs and outputs, and there's enough complexity that it might easily discover edge cases

Reply:
Done in a940eb08aea2701523ff68529684eff12fe361fd.